azure-pipelines.perf-test.yml

```diff
@@ -0,0 +1,61 @@
+pr: none
+
+trigger: none
+
+variables:
+  - group: fabric-variables
+
+pool:
+  vmImage: 'Ubuntu 18.04'
+
+steps:
+  # No need for checkout since we're using build artifacts.
+  - checkout: none
+
+  - template: azure-pipelines.artifacts.yml
+    parameters:
+      artifact: $(artifact)
+      buildId: $(buildId)
+
+  - template: azure-pipelines.tools.yml
+
+  - script: |
+      yarn
+    displayName: yarn
+
+  - script: |
+      yarn just perf-test
+    workingDirectory: apps/perf-test
+    displayName: Perf Test
+
+  - task: AzureUpload@1
+    displayName: Upload Perf Test Result to PR deploy site
+    inputs:
+      SourcePath: 'apps/perf-test/dist'
+      azureSubscription: 'UI Fabric (bac044cf-49e1-4843-8dda-1ce9662606c8)'
+      storage: fabricweb
+      ContainerName: '$web'
+      BlobPrefix: 'pr-deploy-site/$(Build.SourceBranch)/perf-test'
+
+  - task: GithubPRComment@0
+    displayName: 'Post Perf Results to Github Pull Request'
+    inputs:
+      githubOwner: OfficeDev
+      githubRepo: 'office-ui-fabric-react'
+      blobFilePath: '$(Build.SourcesDirectory)/$(PerfCommentFilePath)'
+      status: '$(PerfCommentStatus)'
+      uniqueId: 'perfComment9423'
+
+  - task: GithubPRStatus@0
+    displayName: 'Update Github Pull Request Status'
+    inputs:
+      githubOwner: OfficeDev
+      githubRepo: 'office-ui-fabric-react'
+      githubContext: 'Pull Request Deployed Site'
+      githubDescription: 'Click "Details" to go to the Deployed Site'
+      githubTargetLink: 'http://fabricweb.z5.web.core.windows.net/pr-deploy-site/$(Build.SourceBranch)/'
+
+  - task: DeleteFiles@1
```

Comment:
This is because we're managing our own agents, right? What happens if the job fails before this step?

Reply:
If the job fails before this step, it's kind of nice - that means we can examine the failed build artifacts via ssh if need be. Although we can definitely make this run even if the previous steps failed (delete files task was added before, I'd make that change as part of another change if need be)

Reply:
Does this affect other PR jobs or only any given PR's pipeline? If this could pollute other PRs I definitely feel it should run at all times.

Reply:
I'm adding a `condition: always()` to always clean up